### ISSUE TITLE
Equals

### ISSUE THREAD

**Issue by ehannes** [c7/personnummer#9]
Great gem!

Have you considered implementing an equals method?
```
Personnummer.new('900101-0017') == Personnummer.new('900101-0017')
=> false
```

**Comment by peterhellberg** [comment]
I had not, but I'd probably accept a PR for such a feature. 
(I don't use Ruby in my day to day work anymore)

**Comment by ehannes** [comment]
To bad you are not using Ruby in your work anymore. I find this gem very useful. I will probably use it in a couple of applications. If you are not using it anymore, do you know if the code is still valid and reliable? Anything that needs an update?

If I have time I might submit a PR.

**Comment by peterhellberg** [comment]
I've found that [Go](https://golang.org/) is a better fit for the problems I need to solve these days, that is all.

This project should be functioning as intended as long as there isn't any major changes to the Ruby language or the personnummer system in Sweden. 

I started working on this gem in the summer of 2008, so it has held up pretty well :)

I'm not aware of anything that would require an update, and the library is just some 200 lines of code.
